user-operator/src/main/java/io/strimzi/operator/user/Main.java

```diff
@@ -104,19 +109,19 @@ public static void main(String[] args) {
         return promise.future();
     }
 
-    private static kafka.security.auth.SimpleAclAuthorizer createSimpleAclAuthorizer(UserOperatorConfig config) {
-        log.debug("Creating SimpleAclAuthorizer for Zookeeper {}", config.getZookeperConnect());
-        Map<String, Object> authorizerConfig = new HashMap<>();
-        // The SimpleAclAuthorizer from KAfka requires the Zookeeper URL to be provided twice.
-        // See the comments in the SimpleAclAuthorizer.scala class for more details
-        authorizerConfig.put(kafka.security.auth.SimpleAclAuthorizer.ZkUrlProp(), config.getZookeperConnect());
-        authorizerConfig.put("zookeeper.connect", config.getZookeperConnect());
-        authorizerConfig.put(kafka.security.auth.SimpleAclAuthorizer.ZkConnectionTimeOutProp(), config.getZookeeperSessionTimeoutMs());
-        authorizerConfig.put(kafka.security.auth.SimpleAclAuthorizer.ZkSessionTimeOutProp(), config.getZookeeperSessionTimeoutMs());
+    private static Admin createAdminClient(AdminClientProvider adminClientProvider, UserOperatorConfig config, SecretOperator secretOperations) {
+        Secret clusterCaCert = null;
```

Comment:
I think the naming here is a bit confusing for me. Could we call this `clusterCaCertSecret`?